### PR TITLE
Add sane doc reports

### PR DESCRIPTION
**Pull request by agamm**
Used for the automation that produces docx reports.

**Currently should be in dev mode**.

### CODE REVIEW

**Comment by glicht** [comment]
@agamm build is failing. It seems that sane_src is empty.

**Comment by meirwah** [comment]
@agamm plz note build fails here

File: Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (glicht) [inline Pipfile.lock]: Unnecessary file. Please delete.

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (glicht) [inline Pipfile]: Unnecessary file. Please delete.

**Pull request merged**